### ISSUE TITLE
token-lookup meta not showing after upgrading to 0.6.1 from 0.5.3

### ISSUE THREAD

**Issue by seongkki** [hashicorp/vault#1921]
When performing token-lookup from existing 0.5.3, it shows:
```
Key          	Value
accessor     	179e0a74-a471-01e0-a0c5-563288c8ef97
creation_time	1.474898846e+09
creation_ttl 	3600
display_name 	token-test
id           	73860cb1-59a1-85ed-1eee-fcc144c83edc
meta         	map[key:value]
num_uses     	0
orphan       	false
path         	auth/token/create
policies     	[root]
role
ttl          	3581
```

After upgrading it to 0.6.1, it shows:
```
Key             	Value
accessor        	179e0a74-a471-01e0-a0c5-563288c8ef97
creation_time   	0
creation_ttl    	3600
display_name
explicit_max_ttl	0
id              	73860cb1-59a1-85ed-1eee-fcc144c83edc
meta            	map[key:value]
num_uses        	0
orphan          	false
path            	auth/token/create
policies        	[root]
renewable       	true
ttl             	3491
```

As you could see, `creation_time` is zeroed out, and `display_name` doesn't show up. 
Any idea on the cause of this? It's happening regardless of which version of client side I use,   I have few scripts that replies on display_names and it's stopping me from those operations.

**Comment by jefferai** [comment]
What versions of Vault are your server and client? Were both 0.5.3 and then both upgraded to 0.6.1? Can you verify?

**Comment by seongkki** [comment]
@jefferai , Thanks for your response.
Yes, the server was upgraded to 0.6.1,
and the metadata broke with both 0.5.3 and 0.6.1 Client.

**Comment by jefferai** [comment]
Thanks. We've identified the cause. We'll fix this for 0.6.2.
